v1/chronos.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Chronos
+After=docker.service mesos-master.service
+Requires=docker.service mesos-master.service
+
+[Service]
+Environment=CHRONOS_IMAGE=mesosphere/chronos:chronos-2.3.3-1.0.1.ubuntu1404-mesos-0.22.0-1.0.ubuntu1404
+# NOTE: chronos doesn't need the `zk://`
+Environment=ZOOKEEPER=<ZK_ELB>
+
+Restart=on-failure
```

Comment:
according to http://www.freedesktop.org/software/systemd/man/systemd.service.html , yes.

`Setting this to on-failure is the recommended choice for long-running services, in order to increase reliability by attempting automatic recovery from errors.`